### DISCUSSION TITLE
Surface runs hot and fans don't stop spinning

### DISCUSSION THREAD

**Discussion by AIP21** [badstorm/surface-pro-7-opencore#128] [Q&A]
My surface gets very hot when under a light load (web browsing, document editing, or YouTube) which makes the fans run noisily. Not sure why it does this since the reported CPU usage (in Activity Monitor) is only around 3%. Also the fan remains spinning (albeit slow enough to not be audible) even when the surface is not hot, which wastes battery. This issue does not occur in Windows. When charging (and simultaneously using the device) the issue is made even worse with the surface getting very hot and the fans running super loud. Is there something wrong with the fan curve and is there some way to change the fan curve? 

No other apps are running in the background (no spotlight indexing or photos sync).

Specs:
Surface Pro 7; Intel i7 model; 16gb RAM;

**Comment by badstorm** [comment]
Since the conversation is dated I am closing the issue and moving it as a conversation.
